server/service/src/processors/transfer/requisition/create_response_requisition.rs

```diff
@@ -62,9 +63,15 @@ impl RequisitionTransferProcessor for CreateResponseRequisitionProcessor {
         // TODO link to documentation of how remote authorisation works
         let store_preference =
             get_store_preferences(connection, &record_for_processing.other_party_store_id)?;
+
+        // Check if requisition has items that are part of programs
+        let has_program_items =
+            requisition_has_program_items(connection, &request_requisition.requisition_row.id)?;
+
         // TODO Rework once plugin functionality has been implemented
         let approval_status = if store_preference.response_requisition_requires_authorisation
             && request_requisition.requisition_row.program_id.is_some()
+            || has_program_items
```

Comment:
Hey John, wondering if this should be in brackets?

ie:
 `   let approval_status = if store_preference.response_requisition_requires_authorisation
            && (request_requisition.requisition_row.program_id.is_some()
            || has_program_items)`

Reply:
Yes! Good catch, thanks @fergie-nz. Will apply soon